tapir/coop/forms.py

```diff
@@ -272,38 +272,34 @@ class Meta:
             "share_owner",
             "cancellation_reason",
             "cancellation_date",
-            "coop_buys_shares_back",
-            "willing_to_gift_shares_to_coop",
-            "transfering_shares_to",
+            "resignation_type",
```

Comment:
Are the Textchoices from the model shown in the Form as single checkboxes as before? I didn't find anything on this in the django documentation.

Reply:
They show as a single dropdown instead of several checkboxes.